### PR TITLE
Minor changes to newscategory and download_shortcode, updated core plugins news icons

### PR DESCRIPTION
**Pull request by rica-carv**

### CODE REVIEW

**Comment by CaMer0n** [comment]
Thanks, but you should have done separate pull-request for each plugin/issue.

**Comment by rica-carv** [comment]
Err, sorry, i tried, but i'm a newbie on github matters..... :disappointed:

**Comment by Moc** [comment]
@rica-carv Once you submit a pull-request, all changes that you make will be added onto the same pull-request. You can make additional ones when working with branches. Here is some more info: https://help.github.com/articles/using-pull-requests/

File: e107_plugins/download/download_shortcodes.php [hunk 1]
```diff
@@ -918,9 +918,10 @@ function sc_download_cat_search()
 			
 		$text .= '
               <span class="input-group-btn">
-              <button class="btn btn-default" type="submit" name="s"  value="1">
-              <span class="glyphicon glyphicon-search"></span>
-             </button>
+              <button class="btn btn-default" type="submit" name="s"  value="1">';
+              
+              $text .= '<span class="'.((deftrue('BOOTSTRAP') && deftrue('FONTAWESOME') ? "fa fa" : "glyphicon glyphicon").'-search"></span>';
```
Comment (rica-carv) [inline e107_plugins/download/download_shortcodes.php]: Candidate to recode using the toGlyph function???
Maybe like this:
`$text .= $tp->toGlyph('glyphicon-search');`

**Comment by CaMer0n** [comment]
`$tp->toIcon()` would probably be better.

**Comment by rica-carv** [comment]
@CaMer0n Humm, how would it deal with this: `$tp->toIcon('glyphicon-search')` ???
Soon i can, i'll do some tests...

**Comment by CaMer0n** [comment]
Sure, but it's for backward compatibility. 

`$tp->toIcon('glyphicon-search.glyph'); `

**Comment by rica-carv** [comment]
Hum, got it....

**Comment by rica-carv** [comment]
@Moc I must be real dumb or something, but i can't simply make separate pull requests of the code i've already submitted....
I'm just using the github site, so i can't really figure out how to submit my changes separately, since i've alreay tried make branches, sub branches, sub sub branches, duplicated branches, triplicated branches, and i always get the same, everything goes together...
Sorry, i really would like to help, but this is driving me nuts....
Sorry....

**Comment by Moc** [comment]
@rica-carv Get me on Gitter early next week and I'll to help out. 

By the way, all the duplicating is rarely working. Similar to the templates thing, my advise is to try first how it is supposed to work - e.g. as documented. If that doesn't work, then it is likely a bug or something you're doing wrong. In that case, stop and ask for help :)

**Comment by rica-carv** [comment]
@moc Tried once more right now, https://github.com/e107inc/e107/pull/1550

**Comment by Moc** [comment]
@rica-carv Yes, I think that looks good! :+1: Well done

**Comment by rica-carv** [comment]
Hurrah.... preparing next ones....
Got to clean all changes and made them again... ufff....

**Comment by Moc** [comment]
Alright. Perfect.

You can improve by making the pull-request title shorter and by including the issue number you are referring to e.g. (#XXX). Github will automatically make references so it's really easy to track all the changes and history log.

**Comment by rica-carv** [comment]
Err, ok, gonna try with the next one... thanks....

**Comment by Moc** [comment]
See #1526 for an example :) (I just changed the title)

**Pull request closed without merging**